gravitee-apim-console-webui/package.json

```diff
@@ -15,7 +15,7 @@
     "@angular/upgrade": "12.2.3",
     "@asyncapi/web-component": "1.0.0-next.15",
     "@fontsource/libre-franklin": "4.4.5",
-    "@gravitee/ui-components": "3.22.0",
+    "@gravitee/ui-components": "^3.24.4",
```

Comment:
you should use pinned version

Reply:
yes, I don't understand why npm is adding it back to me like this